docs/variant-specific/pink.mdx

```diff
@@ -253,25 +256,69 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
+- It is possible for a clue to be both a _Bubblegum Finesse_ and a _Bubblegum Prompt_ at the same time.
+  - In the previous example, we can modify Donald's hand such that he only holds the pink 4 (instead of holding both the pink 4 and the pink 3).
+  - In the previous example, Cathy discarded, but in the modified example, Cathy would know that the pink 4 was promised to be playable. And Cathy would not see anyone else having the pink 3, so Cathy would have to respond by playing a pink 3 from her _Finesse Position_.
 
-### The Bubblegum Bluff
+### The Bubblegum Bluff (Standard Form)
 
 - Similar to a _Bubblegum Finesse_, it is also possible to perform a _Bubblegum Bluff_.
 - For example, in a 3-player game:
   - It is the first turn and nothing is played on the stacks.
   - Alice clues number 3 to Cathy, touching a pink 2 on slot 5.
   - Bob knows that Alice is violating _Pink Promise_. This must mean that he is supposed to play his slot 3 card.
   - Bob blind-plays slot 3 and it is a red 1.
-  - Cathy knows that if this was a _3 Bluff_, then Bob would have played his _Finesse Position_. So it cannot be a _3 Bluff_.
+  - Cathy knows that if this were a _3 Bluff_, then Bob would have played his _Finesse Position_. So it cannot be a _3 Bluff_.
   - Thus, Cathy knows that this must be a _Bubblegum_ clue. Since red 1 is not a pink card and Alice was in _Bluff Seat_ for Bob, this must be a _Bubblegum Bluff_.
   - Cathy marks her slot 5 card as the one-away-from-playable pink card, the pink 2.
 
 <BubblegumBluff />
 
-- _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Double Finesse_ with one part _Bubblegum Finesse_ and one part normal _Finesse_.
+- _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Bubblegum Finesse_.
 - Note that it is impossible to _3 Bluff_ and _Bubblegum Bluff_ at the same time.
 
+### The Bubblegum Bluff (Two-Away-From-Playable Form)
+
+- Normally, _Bubblegum Bluffs_ are always performed by touching _one-way-from-playable_ pink cards.
+- Rarely, it is possible to perform a _Bubblegum Bluff_ on a _two-away-from-playable_ pink card. However, this can only be done in a specific situation where, due to the circumstances of the clue, it would be impossible for the clued card to be _one-way-from-playable_.
+- For example, in a 3-player game:
+  - It is the first turn and nothing is played on the stacks.
+  - Alice clues number 2 to Cathy, touching a pink 3 on slot 3 as a _Play Clue_. (The pink 3 is currently _two-away-from-playable_.)
+  - Bob knows that Alice is violating _Pink Promise_.
+  - If Bob blind-plays his _Finesse Position_ card and it successfully plays, it is guaraunteed to be one of the 1s. If that happened, then Cathy would mark her clued card as the matching 2 and will go on to misplay it.
```

Comment:
(oops, I need to fix this typo... will do that sometime before submission.)

Reply:
what is the typo?

Reply:
some check was failing: "guaranteed" only has one "u"